### DISCUSSION TITLE
Corruption of files when adding local time signatures

### DISCUSSION THREAD

**Discussion by mozbadel** [musescore/MuseScore#16817] [Q&A]
For some years I have been trying to complete my transcription of Mother Goose but failing because of corruptions when using time signature changes for some (but not all) instruments.

I have also raised this issue in the support section of the forum on Musescore.org and I have attached four files on there with subscripts a through d, showing:

a) 2/4 successfully added to Celesta at rehearsal mark 17
b) 2/4 successfully added to 2nd Violins at rehearsal mark 17
c) 6/8 successfully added to Violas (Altos) at rehearsal mark 17
d) 6/8 causing corruption to file when added to 2nd Violins at rehearsal mark 18

There have been no notes added to the files at all as yet.

I would be grateful if you could investigate and look to fix this problem which prevents Musescore being a serious option for music transcription
Many thanks.

**Comment by ghost** [comment]
I have made this into an issue: https://github.com/musescore/MuseScore/issues/16825

**Comment by mozbadel** [comment]
Thanks! Marc Sabatella has given me a workaround but it would be good to get the problem fixed.
